lldb/source/Commands/CommandObjectDWIMPrint.cpp

```diff
@@ -130,7 +130,20 @@ void CommandObjectDWIMPrint::DoExecute(StringRef command,
     }
   };
 
-  // First, try `expr` as the name of a frame variable.
```

Comment:
1. I'd prefer to leave the "first", "second", "third" comments. I want to be explicit that there's an order of attempts.
2. You'll need to rebase, I made a change last week that introduce some comments and code that you may need to incorporate.